app/views/home/businesses.html.erb

```diff
@@ -21,49 +21,49 @@
 </div>
 
 <div class="container-lg py-6 py-3 px-3 text-center">
-  <h2 class="alt-h2">Your business depends on it</h2>
+  <h2 class="alt-h2"><%=t "Your business depends on it" %></h2>
 
   <p class="alt-lead text-gray mx-auto text-center mb-6">
-    Companies need open source software to run effectively.
+  <%=t "Companies need open source software to run effectively" %>
   </p>
 </div>
 <div class="container-lg mx-auto gut-lg text-center">
   <div class="mb-6 col-md-6 float-md-left">
     <%= image_tag("improve-technology.svg", class: "img-responsive mb-3", height: "100px") %>
-    <h3 class="alt-h3 mb-2">Improve the technology you rely on</h3>
+    <h3 class="alt-h3 mb-2"><%=t "Improve the technology you rely on" %></h3>
     <p class="text-gray">
-      Direct participation in open source gives your organization a voice in the software you depend on.
+      <%=t "Direct participation in open source" %>
```

Comment:
There is no reason to use full length lkey for translator because translator can see en.yml value for translation not key.  
Redmine that is famouse rails app use a shot key like  'ticket.title' for key. It is a best plactise to use short key for app. I  don't know a best plactise for content localisation like here.